src/prx/main.py

```diff
@@ -361,12 +361,12 @@ def format_flat_rows(row):
     #  the same satellite and the same time of emission
     sat_specific = sat_states[
         sat_states.columns.drop(
-            ["observation_type", "group_delay_m", "time_of_reception_in_receiver_time"]
+            ["observation_type", "sat_instrumental_delay_m", "time_of_reception_in_receiver_time"]
         )
     ].drop_duplicates(subset=["satellite", "time_of_emission_isagpst"])
     # Group delays are signal-specific, so we merge them in separately
     code_specific = sat_states[
-        ["satellite", "observation_type", "time_of_emission_isagpst", "group_delay_m"]
+        ["satellite", "observation_type", "time_of_emission_isagpst", "sat_instrumental_delay_m"]
```

Comment:
would `broadcast_code_bias` be an appropriate name here?

Reply:
There are many possible models related to this parameter: differential code bias or total/timing/broadcast group delay.
Those terms are referred to as satellite instrumental delays in the ESA book, hence the proposal.
I think in the long term, we will have to account for satellite code and phase bias. We could then go for sat_code_bias_m for the moment?